src/software/logger/network_sink.cpp

```diff
@@ -0,0 +1,32 @@
+#include "software/logger/network_sink.h"
+
+#include <experimental/filesystem>
+
+#include "shared/proto/robot_log_msg.pb.h"
+#include "software/constants.h"
+
+NetworkSinc::NetworkSinc(int channel, const std::string& interface, int robot_id)
+{
+    robot_id_ = robot_id;
```

Comment:
Generally if one variable has a `_` suffix all variables do. In this case we can probably instead use `this.robot_id` or an initialization list